bench-tps/src/bench.rs

```diff
@@ -144,13 +146,19 @@ pub fn do_bench_tps<T>(
         // this seems to be faster than trying to determine the balance of individual
         // accounts
         let len = tx_count as usize;
+        let mut new_blockhash = client.get_recent_blockhash().unwrap();
+        while new_blockhash == blockhash {
+            sleep(Duration::from_millis(100));
+            new_blockhash = client.get_recent_blockhash().unwrap();
```

Comment:
@pgarg66 I'm seeing this panic on testnet. Unsure what it's supposed to do if we can't get a recent blockhash lol

Reply:
Let's debug it today.